designer/client/src/actions/notificationActions.tsx

```diff
@@ -12,7 +13,10 @@ export function success(message: string): Action {
     });
 }
 
-export function error(message: string): Action {
+// TODO This method is incomplete. I've added error and showErrorText params to show what is missing.
```

Comment:
I'm not sure what action should be performed in case of this TODO, but we have epic in [Jira](https://touk-jira.atlassian.net/browse/NU-1702) for error handling, maybe you can add a task for it with a description of the problem and get rid of this comment for now and remove unused params?

Reply:
If you are working on it then ok. I am still not sure it is good idea to leave it unfixed - I leave it though, hoping it is going to be made all right :)